### PR TITLE
Create Slider.js component with test

### PR DESCRIPTION
**Pull request by akilhylton**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

**Code formatting**

**Closing issues**

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Comment by akilhylton** [comment]
[Image of the slider component](https://i.imgur.com/pGabN6h.jpg)

**Comment by SomeMoosery** [comment]
LGTM! Appreciate the screenshot 👍

File: client/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "prettier": "@blockstack/prettier-config",
   "dependencies": {
     "@fortawesome/free-brands-svg-icons": "^5.12.1",
+    "@material-ui/core": "^4.9.7",
```
Comment (SomeMoosery) [inline client/package.json]: Just curious, does using Material-UI introduce a second UI framework along with React Bootstrap? Or is this just a library of UI components and the two can work in tandem? If the former I feel like we should implement this slider with React Bootstrap to stay consistent, but if not then LGTM

Comment (akilhylton) [reply client/package.json]: React Bootstrap doesn't have a slider library but it might be a good idea to migrate to Material-UI later down the line.

Comment (SomeMoosery) [reply client/package.json]: Sounds good. Yeah, I've only worked with Material-UI before so I'm definitely open to discussing a switch / refactor

**Comment by akilhylton** [comment]
closes #82

**Pull request merged**